src/views/Cookies/Cookies.module.css

```diff
@@ -0,0 +1,53 @@
+.mainWrap {
+  place-content: center;
+  flex: 1;
+  padding-top: 80px;
+  padding-bottom: 80px;
+  overflow-y: auto;
+}
+
+.content {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 16px;
+  line-height: 24px;
+  box-sizing: content-box;
+  padding-left: 14px;
+  padding-right: 14px;
+}
+
+.content blockquote {
+  margin-block-start: 2em;
+  margin-block-end: 2em;
+}
+
+.content blockquote h3 {
+  margin-block-end: 0em;
+}
+
+.content blockquote p {
+  margin-block-start: 0.3em;
+  padding-left: 1em;
+}
+
+a,
+.link {
+  color: var(--accent-strong);
+  text-decoration: none;
+}
+
+.content table {
+  margin-top: calc(var(--double-unit) * 4);
```

Comment:
maybe another candidate to extend our offset library values

Reply:
can't find where such values also used 
![image](https://user-images.githubusercontent.com/6587585/205936738-a78992d4-8aff-48df-a54e-32c13f5c6860.png)

Reply:
i mean it seems like another arbitrary value